root/static/scripts/edit/URLCleanup.js

```diff
@@ -455,11 +455,11 @@ const CLEANUPS = {
       return null;
     },
     validate: function (url) {
-      return /^https:\/\/www\.amazon\.(com|ca|co\.uk|fr|at|de|it|co\.jp|jp|cn|es|in|com\.br|com\.mx|com\.au)\//.test(url);
+      return /^https:\/\/www\.amazon\.(com|ca|co\.uk|fr|ae|at|de|it|sg|co\.jp|jp|cn|es|in|nl|com\.br|com\.mx|com\.au|com\.tr)\//.test(url);
     },
   },
   'amazonmusic': {
-    match: [new RegExp('^(https?://)?music\\.amazon\\.(com|ca|co\\.uk|fr|at|de|it|co\\.jp|jp|cn|es|in|com\\.br|com\\.mx|com\\.au)/(albums|artists)', 'i')],
+    match: [new RegExp('^(https?://)?music\\.amazon\\.(com|ca|co\\.uk|fr|ae|at|de|it|sg|co\\.jp|jp|cn|es|in|nlcom\\.br|com\\.mx|com\\.au|com\\.tr)/(albums|artists)', 'i')],
```

Comment:
Missing `|` broke Brazilian Amazon store.
```suggestion
    match: [new RegExp('^(https?://)?music\\.amazon\\.(com|ca|co\\.uk|fr|ae|at|de|it|sg|co\\.jp|jp|cn|es|in|nl|com\\.br|com\\.mx|com\\.au|com\\.tr)/(albums|artists)', 'i')],
```

Reply:
Whoops :) Fixed!